samples/topics.js

```diff
@@ -157,16 +156,19 @@ async function publishBatchedMessages(
   // Publishes the message as a string, e.g. "Hello, world!" or JSON.stringify(someObject)
   const dataBuffer = Buffer.from(data);
 
-  const [messageId] = await pubsub
-    .topic(topicName, {
-      batching: {
-        maxMessages: maxMessages,
-        maxMilliseconds: maxWaitTime,
-      },
-    })
-    .publish(dataBuffer);
-  console.log(`Message ${messageId} published.`);
+  const batchPublisher = pubsub.topic(topicName, {
+    batching: {
+      maxMessages: maxMessages,
+      maxMilliseconds: maxWaitTime,
+    },
+  });
 
+  for (let i = 1; i < 11; i++) {
```

Comment:
nit: slightly prefer `i = 0; i < 10` here, but logic doesn't change so feel free to ignore.